src/Atmos/Parameterizations/TurbulenceConvection/EDMF/EDMF.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    Eddy-Diffusivity-Mass-Flux (EDMF)
+
+Module for solving the sub-grid-scale equations
+using the Eddy-Diffusivity-Mass-Flux (EDMF) model.
+"""
+module EDMF
+
+using Pkg
+haspkg(pkgname::String) = haskey(Pkg.installed(), pkgname)
+@static if haspkg("Plots")
+  using Plots
+end
```

Comment:
Use Requires instead of haspkg.

Reply:
Can we somehow revive `haspkg`? I'd prefer avoiding `Requires` hashes sprinkled everywhere. @simonbyrne

Reply:
Requires is not functionally equivalent to `haspkg` (it cannot be used inside functions). These will be removed eventually, but I'd prefer to keep this functionality for the time being.

Reply:
No, `haspkg` doesn't really work on Julia 1.3, or play nice with precompilation.